.travis.yml

```diff
@@ -1,9 +1,23 @@
-dist: trusty
-sudo: false
-
 language: python
-python:
-  - "3.6"
+
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.6
+      dist: trusty
+      env: TOXENV=py36
+      sudo: false
+    - python: 3.7
+      # from https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442
+      dist: xenial
+      env: TOXENV=py37
+      sudo: true
+
+install:
+    - pip install -e ".[dev]"
```

Comment:
You only need to install `tox` here. Whenever it's run at line 20, it'll install automatically it's own virtual env with all the dependencies that you set up in https://github.com/mozilla-releng/shipitapi/pull/8/files#diff-b91f3d5bd63fcd17221b267e851608e8R15